README.Rmd

```diff
@@ -59,13 +59,20 @@ data_file_complex <- system.file(
    "extdata", "sample_data_file_large.txt", package = "maxent.ot"
 )
 
+
 # Fit weights to both data sets with simple regularization
 simple_model <- optimize_weights(data_file_simple, mu_scalar=0, sigma_scalar=10)
 complex_model <- optimize_weights(data_file_complex, mu_scalar=0, sigma_scalar=10)
 
+
 # Examine predicted probabilities of each model
+# Calculate predicted probabilities of each model
 simple_predictions <- predict_probabilities(data_file_simple, simple_model$weights)
```

Comment:
Why don't we just change these lines to be

```
predict_probabilities(data_file_simple, simple_model$weights)
predict_probabilities(data_file_complex, complex_model$weights)
```
and delete the lines with `$predictions` below. This will display both the log likelihood and predictions, and keep the code a little simpler here.

Reply:
I think you need to re-knit the README.md file from the Rmd file.